parser/generator.go

```diff
@@ -162,6 +162,7 @@ func (g *Generator) GenerateDockerfile() error {
 
 	for _, phase := range phases {
 		if len(phase.commands) > 0 {
+			bufferRun.WriteString("source /.bzkenv\n")
```

Comment:
Not a blocker, but this should not be generated in the very first iteration, since `.bzkenv` doesn't even exist yet.

As it is now, we end up with this (notice the very first `source`):

```shell
#!/bin/bash
source /.bzkenv
/bazooka/bazooka_install.sh
rc=$?
if [[ $rc != 0 ]] ; then
    exit 42
fi
source /.bzkenv
/bazooka/bazooka_script.sh
exitCode=$?
source /.bzkenv
exit $exitCode
```